Assets/General/Scripts/Managers/UniversalManagers/MainUniversalManagers/AframaxSceneManager.cs

```diff
@@ -25,8 +25,13 @@ public class AframaxSceneManager : MainUniversalManagerFramework
     [field: SerializeField] public int MainMenuSceneIndex { get; private set; }
 
     [field: SerializeField] public int DeathScreenSceneIndex { get; private set; }
+
     [field: SerializeField] public int EndScreenSceneIndex { get; private set; }
 
+    [field: SerializeField] public int SettingsSceneIndex { get; private set; }
+
+    public int _lastSceneIndex;
```

Comment:
should be private

Reply:
how will other things (like the settings menu) look at it though?